src/Modalizer.ts

```diff
@@ -654,7 +656,13 @@ export class ModalizerAPI implements Types.ModalizerAPI {
         const parts = this._parts;
         const visibleElements: HTMLElement[] = [];
         const hiddenElements: HTMLElement[] = [];
-        const alwaysAccessibleElements: HTMLElement[] = [];
+        const alwaysAccessibleSelector = this._alwaysAccessibleSelector;
+        const alwaysAccessibleElements: HTMLElement[] = alwaysAccessibleSelector
```

Comment:
Is it safe to pull this on modalizer creation only? what if there are more always accessible elements added after the modalizer is created?